easybuild/easyconfigs/j/jupyterlab/jupyterlab-1.2.4-CrayGNU-19.10-cuda-10.1-batchspawner.eb

```
@@ -1,18 +1,12 @@
 # @author: robinson (omlins and hvictor for IJulia)
 
-easyblock = 'Bundle'
+easyblock = 'PythonBundle'
 
 name = 'jupyterlab'
 version = '1.2.4'
-_cudaversion = '10.1'
-versionsuffix = '-cuda-%s-batchspawner' % _cudaversion
+versionsuffix = '-cuda-%(cudashortver)s-batchspawner'
 
-_py_maj_ver = '3'
 _py_min_ver = '6'
```

```suggestion
_pyminver = '6'
```
Better change this, because I think `_pyminver` is what we are replacing in the ExperimentalEB